diff --git a/chrome/browser/resources/side_panel/history_clusters/app.html.ts b/chrome/browser/resources/side_panel/history_clusters/app.html.ts
index fd7dd69ec6fd73d7e16f80aa6aeb3460a16d1d41..f932ec3eda00d2e0d5e39be82c510b012b733437 100644
--- a/chrome/browser/resources/side_panel/history_clusters/app.html.ts
+++ b/chrome/browser/resources/side_panel/history_clusters/app.html.ts
@@ -20,12 +20,6 @@ export function getHtml(this: HistoryClustersAppElement) {
 ${this.enableHistoryEmbeddings_ ? html`
 <div id="historyEmbeddingsDisclaimer">
   $i18n{historyEmbeddingsDisclaimer}
-  <a id="historyEmbeddingsDisclaimerLink" href="#"
-      aria-describedby="historyEmbeddingsDisclaimer"
-      @click="${this.onHistoryEmbeddingsDisclaimerLinkClick_}"
-      @auxclick="${this.onHistoryEmbeddingsDisclaimerLinkAuxclick_}">
-    $i18n{learnMore}
-  </a>
 </div>
 ` : ''}
 <div id="embeddingsScrollContainer"
