diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index bbe0fe895fe3077b7f4992d324e02d0dab8001c6..ac479cef659a7793b74d382dc261c3c28ee5fb39 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/extensions/resources.gni")
 import("//build/config/chromebox_for_meetings/buildflags.gni")
 import("//build/config/features.gni")
 import("//build/config/ozone.gni")
@@ -651,6 +652,7 @@ source_set("extensions") {
     "//ui/native_window_tracker",
     "//ui/shell_dialogs",
   ]
+  deps += brave_extensions_resources
   public_deps = [
     "//chrome/browser/extensions/api/messaging",
     "//chrome/browser/media/webrtc",
@@ -694,7 +696,7 @@ source_set("extensions") {
       "//chrome/browser/ui/android/tab_model",
     ]
 
-    if (enable_desktop_android_extensions) {
+    if (enable_extensions_core) {
       sources += [
         # These files are temporary workarounds to allow for a lightweight
         # extensions runtime in desktop-android builds. They should be removed
@@ -2053,6 +2055,7 @@ source_set("unit_tests") {
     "//base",
     "//base/test:test_config",
     "//base/test:test_support",
+    "//brave/browser/extensions",
     "//chrome/browser:browser_process_impl",
     "//chrome/browser:global_features",
     "//chrome/browser/background",
