diff --git a/chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.cc b/chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.cc
index 93015c12f79cbbe1c2f127f2c013229ba8c674c7..36c2f15f201c446af4ba89d52935d10a02b49d71 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.cc
@@ -77,7 +77,7 @@ namespace {
 
 // Max width of a menu. There does not appear to be an OS value for this, yet
 // both IE and FF restrict the max width of a menu.
-const int kMaxMenuWidth = 400;
+const int kMaxMenuWidth = 500;
 
 size_t GetSubmenuChildCount(const MenuItemView* menu) {
   return menu->HasSubmenu() ? menu->GetSubmenu()->children().size() : 0;
