diff --git a/chrome/browser/chrome_content_browser_client.cc b/chrome/browser/chrome_content_browser_client.cc
index 99db6a609baea3baefc56e3962fd10548fa0e58f..f243c2c374e55f64a6c6cf89bbd98f883db386d2 100644
--- a/chrome/browser/chrome_content_browser_client.cc
+++ b/chrome/browser/chrome_content_browser_client.cc
@@ -9060,6 +9060,7 @@ base::FilePath ChromeContentBrowserClient::GetChildProcessPath(int flags) {
 
 bool ChromeContentBrowserClient::ShouldUseFirstPartyStorageKey(
     const url::Origin& origin) {
+  if (BraveShouldUseFirstPartyStorageKey(origin)) return true;
 #if BUILDFLAG(ENABLE_EXTENSIONS_CORE)
   return origin.scheme() == extensions::kExtensionScheme;
 #else
