diff --git a/chrome/browser/extensions/chrome_extensions_browser_client.cc b/chrome/browser/extensions/chrome_extensions_browser_client.cc
index d47c2acb103a67155f2b59ee34107633ba8665b6..3ea0cc96f2697816e07a430d291b900c375589b6 100644
--- a/chrome/browser/extensions/chrome_extensions_browser_client.cc
+++ b/chrome/browser/extensions/chrome_extensions_browser_client.cc
@@ -277,7 +277,7 @@ bool RegisterTransformers() {
 ChromeExtensionsBrowserClient::ChromeExtensionsBrowserClient()
     : resource_manager_(
           std::make_unique<ChromeComponentExtensionResourceManager>()),
-      api_client_(std::make_unique<ChromeExtensionsAPIClient>()),
+      api_client_(std::make_unique<BraveExtensionsAPIClient>()),
       event_router_forwarder_(base::MakeRefCounted<EventRouterForwarder>()) {
   AddAPIProvider(std::make_unique<CoreExtensionsBrowserAPIProvider>());
   AddAPIProvider(std::make_unique<ChromeExtensionsBrowserAPIProvider>());
@@ -682,6 +682,10 @@ void ChromeExtensionsBrowserClient::AttachExtensionTaskManagerTag(
 scoped_refptr<update_client::UpdateClient>
 ChromeExtensionsBrowserClient::CreateUpdateClient(
     scoped_refptr<update_client::Configurator> configurator) {
+  if (base::FeatureList::IsEnabled(
+    extensions_features::kBraveSequentialExtensionsUpdate)) {
+    return update_client::SequentialUpdateClientFactory(configurator);
+  }
   return update_client::UpdateClientFactory(configurator);
 }
 
@@ -697,7 +701,7 @@ ChromeExtensionsBrowserClient::CreateUpdateClientConfigurator(
       override_url = update_url;
     }
   }
-  return ChromeUpdateClientConfig::Create(context, override_url);
+  return BraveUpdateClientConfig::Create(context, override_url);
 }
 
 std::unique_ptr<ScopedBrowserContextKeepAlive>
