diff --git a/components/sync/base/features.cc b/components/sync/base/features.cc
index 4d909e2968a54f2ab3f04570c28fe242ad4e9a06..4a74c96c4a30bd4383ab74d9ae0ac5f632116d94 100644
--- a/components/sync/base/features.cc
+++ b/components/sync/base/features.cc
@@ -89,6 +89,7 @@ BASE_FEATURE(kReplaceSyncPromosWithSigninPromosNewSignin,
 );
 
 bool IsReplaceSyncPromosWithSignInPromosEnabled() {
+  if ((true)) return false;
   return base::FeatureList::IsEnabled(kReplaceSyncPromosWithSignInPromos) ||
          base::FeatureList::IsEnabled(
              kReplaceSyncPromosWithSigninPromosNewSignin);
@@ -118,7 +119,7 @@ DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAccountSettings);
 DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuable);
 #endif
 
-DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuableMetadata);
+BASE_OVERRIDDEN_FEATURE(kSyncAutofillValuableMetadata, base::FEATURE_DISABLED_BY_DEFAULT);
 
 DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncWalletFlightReservations);
 
