diff --git a/extensions/browser/BUILD.gn b/extensions/browser/BUILD.gn
index 43b44ac41ab5c357dc65f31f58fdfe7b0950cf8c..fdc6035662de40d638b3059e6d9abe12998de275 100644
--- a/extensions/browser/BUILD.gn
+++ b/extensions/browser/BUILD.gn
@@ -10,7 +10,7 @@ import("//rlz/buildflags/buildflags.gni")
 import("//testing/libfuzzer/fuzzer_test.gni")
 import("//third_party/protobuf/proto_library.gni")
 
-if (enable_desktop_android_extensions) {
+if (is_android && enable_extensions_core) {
   import("//build/config/android/rules.gni")
 }
 
@@ -1325,7 +1325,7 @@ source_set("unit_tests") {
   configs += [ "//build/config/compiler:no_exit_time_destructors" ]
 }
 
-if (enable_desktop_android_extensions) {
+if (is_android && enable_extensions_core) {
   java_cpp_enum("extension_action_enums") {
     sources = [ "extension_action.h" ]
   }
