diff --git a/ui/views/view.h b/ui/views/view.h
index 242fc32c0b862217706bab93e380df5e8f0441b5..96f93be79d3f7e248f84fc027368845f7710511d 100644
--- a/ui/views/view.h
+++ b/ui/views/view.h
@@ -321,6 +321,7 @@ class VIEWS_EXPORT View : public ui::LayerDelegate,
     FRIEND_TEST_ALL_PREFIXES(WebViewUnitTest, CrashedOverlayView);
 
     OwnedByClientPassKey() = default;
+    BRAVE_VIEW_OWNED_BY_CLIENT_PASS_KEY
   };
 
   using PassKey = base::NonCopyablePassKey<View>;
